komga-webui/src/components/DropZone.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <label class="drop-zone" v-cloak @drop.prevent="dropHandler" @dragover.prevent>
+    <span class="file-input">Choose an image</span> - drag and drop
+    <input hidden aria-hidden="true" type="file" accept="image/*" multiple @change="dropHandler" >
+  </label>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+
+export default Vue.extend({
+  name: 'DropZone',
+  data() {
+    return {
+      files: [] as File[],
+    }
+  },
+  watch: {
+    files: function (value) {
+      this.$emit('onInputChange', value)
```

Comment:
Should use kebab-case for events, see https://vuejs.org/v2/guide/components-custom-events.html#Event-Names